### DISCUSSION TITLE
Istio Egress gateway

### DISCUSSION THREAD

**Discussion by jessyjes** [istio/istio#48701] [Q&A]
Hello everybody
I would like to have some information about Egress gateway.
We have a kubernetes cluster and we would like to dedicate one or more nodes to be allowed to reach external website.
The Idea is to allow pods from other nodes of the clusters that need to reach external urls, to be able  to do that only because there is a specific configuration that route that kind of traffic to the dedicated nodes configured with istio Egress gateway for that
Is it possible to set such a configuration with istio ?
Or do i need to set another dedicated kubernetes cluster for istio Egress gateway ?

Thanks for your answers

**Comment by husnialhamdani** [comment]
The Istio Egress gateway itself would be enough, isn't it?
and add nodeSelector to shedule that gateway on the dedicated node you want.

And [this setting](https://istio.io/latest/docs/tasks/traffic-management/egress/egress-gateway/#apply-kubernetes-network-policies) will enforce to not bypass egress gateway.

**Reply by jessyjes** [reply]
Oh yes !
Nodeselector , i haven't think to that
Thanks
